src/lib/mixer_module/mixer_module.hpp

```diff
@@ -226,6 +252,7 @@ class MixingOutput : public ModuleParams
 
 	bool _safety_off{false}; ///< State of the safety button from the subscribed _safety_sub topic
 	bool _throttle_armed{false};
+	bool _ignore_lockdown{false}; ///< if true, ignore the _armed.lockdown flag (for HIL outputs)
```

Comment:
Why do we do lockdown if when the ignore it again? That seems weird.

Reply:
Yes, why can't pwm_out_sim be oblivious to HIL?